src/vs/workbench/services/actions/common/menusExtensionPoint.ts

```diff
@@ -343,6 +343,11 @@ const apiMenus: IAPIMenu[] = [
 		id: MenuId.TestItemGutter,
 		description: localize('testing.item.gutter.title', "The menu for a gutter decoration for a test item"),
 	},
+	{
+		key: 'testing/pythonConfig',
```

Comment:
This isn't just for Python, other extensions can use it as well. It should be named more generically.

Perhaps `testing/profiles/context`